### PR TITLE
DRY up selectize users helpers

### PR DESCRIPTION
**Pull request by bear454**
(cherry picked from commit eb38ddc6c761d4920b35499690fd5681f0ea5385)

**Checklist**

- [x] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [x] My branch is up-to-date with the upstream `master` branch.
- [x] The tests pass locally with my changes.

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- `selectize...` helpers had duplicate code.

**Changes proposed in this pull request:**

- DRY up `selectize...` helpers.

### CODE REVIEW

File: app/views/booths/_form.html.haml [hunk 1]
```diff
@@ -24,7 +24,6 @@
   :javascript
     $(document).ready(function() {
       $('#booth_responsible_ids').selectize({
-        plugins: ['remove_button'],
-        minItems: 2
```
Comment (Ana06) [inline app/views/booths/_form.html.haml]: how is that related? :confused:

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -119,18 +119,44 @@ def speaker_links(event)
   end
 
   def speaker_selector_input(form)
-    users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
-    form.input :speakers, as: :select,
-                          collection: options_for_select(users.map {|user| ["#{user[1]} (#{user[2]}) #{user[3]}", user[0]]}, @event.speakers.map(&:id)),
-                          include_blank: false, label: 'Speakers', input_html: { class: 'select-help-toggle', multiple: 'true' }
+    # users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
```
Comment (Ana06) [inline app/helpers/application_helper.rb]: shouldn't those comments be removed as well? :confused:

File: app/views/booths/_form.html.haml (hunk 1 above)
Comment (bear454) [reply app/views/booths/_form.html.haml]: Nice catch, thanks!

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (bear454) [reply app/helpers/application_helper.rb]: Yes... sorry I forgot to remove those.

**Comment by bear454** [comment]
Thanks!
